documentation/puppetdb-faq.markdown

```diff
@@ -10,12 +10,6 @@ canonical: "/puppetdb/latest/puppetdb-faq.html"
 [maintaining_tuning]: ./maintain_and_tune.html
 [low_catalog_dupe]: ./trouble_low_catalog_duplication.html
 
-## Can I migrate my data from ActiveRecord storeconfigs or from an existing PuppetDB to a new instance?
-
-Yes.  At this time, you can only migrate exported resources from ActiveRecord, and
-you can migrate catalogs from an existing PuppetDB.  For more information, see
-[Migrating Data][migrating] for more information.
```

Comment:
we could even put a note in there about referring to previous documentation, in case they're trying to come from storeconfigs all the way up to PDB 4.